code/lib/cli/src/automigrate/types.ts

```diff
@@ -12,7 +12,8 @@ export interface RunOptions<ResultType> {
 
 export interface Fix<ResultType = any> {
   id: string;
+  promptOnly?: boolean;
   check: (options: CheckOptions) => Promise<ResultType | void>;
   prompt: (result: ResultType) => string;
-  run: (options: RunOptions<ResultType>) => Promise<void>;
+  run?: (options: RunOptions<ResultType>) => Promise<void>;
```

Comment:
for future improvement:

This might be better as a union of 2 types, where 1 has `promptOnly: true`, and therefore has no `run` method, and another with it set to `false` and thus MUST have a `run` method.

Reply:
Agreed! Let's do it later